annotation_store/views.py

```diff
@@ -0,0 +1,118 @@
+from django.conf import settings
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_http_methods
+from django.contrib.auth.decorators import login_required
+
+from target_object_database.models import TargetObject
+from hx_lti_initializer.models import LTICourse 
+from hx_lti_assignment.models import Assignment
+from hx_lti_initializer.utils import retrieve_token
+from hx_lti_initializer import annotation_database
+from store import AnnotationStore
+
+import json
+import requests
+import urllib
+import logging
+
+logger = logging.getLogger(__name__)
+
+@require_http_methods(["GET"])
+def api_root(request):
+    return AnnotationStore.from_settings(request).root()
+
+@require_http_methods(["GET"])
+def search(request):
+    return AnnotationStore.from_settings(request).search()
+
+@csrf_exempt
+@require_http_methods(["POST"])
+def create(request):
+    return AnnotationStore.from_settings(request).create()
+
+@csrf_exempt
+@require_http_methods(["PUT"])
```

Comment:
@arthurian Did you change it in annotator to send a PUT instead of POST? or has it always sent a PUT? I saw that Mirador sends it as post so it's hitting a 405 error and update is not working.

Reply:
I don't recall changing anything on the annotator side of things, but it appears that there is a difference between how updates are handled on text vs images: text annotation updates send `PUT` and mirador/image updates send `POST`. I'll fix that so it accepts both.

Reply:
Perfect! Once Rashmi is back we should talk to her about switching this to PUT since it makes more sense and it aligns with where the catch is going.